sass/components/_sword.scss

```diff
@@ -0,0 +1,51 @@
+.sword {
+  height: 50px;
+  width: 1280px;
+  margin: auto;
+}
+.sword-wrap {
+  display: flex;
+  align-items: center;
+}
+
+.sword-hilt {
+  position: relative;
+  width: 299px;
+  height: 50px;
+  display: flex;
+  align-items: center;
+}
+.sword-hilt-image {
+  width: 299px;
+  position: absolute;
+  z-index: -20;
```

Comment:
Kolejnośś, najpierw position ;)

Reply:
Różne są trendy, ale wszystkie podobne:
Proponuję
`content
z-index
overflow
display
position
itd`

Reply:
Różne są podejścia i konwencje, to że ja robię inaczej, nie oznacza, że robię źle